Userland/Applications/CharacterMap/CharacterSearchWidget.cpp

```diff
@@ -96,9 +95,9 @@ void CharacterSearchWidget::search()
 
         model.add_result({ code_point, builder.to_deprecated_string(), move(display_name) });
 
-        // Stop when we reach 500 results.
+        // Stop when we reach 250 results.
         // This is already too many for the search to be useful, and means we don't spend forever recalculating the column size.
-        if (model.row_count({}) >= 500)
+        if (model.row_count({}) >= 250)
```

Comment:
This bit should be squashed to the previous commit?

Reply:
:man_facepalming: